### DISCUSSION TITLE
关于侧边栏配置

### DISCUSSION THREAD

**Discussion by QrSn114** [YunYouJun/hexo-theme-starry#248] [General 大局为重]
%%%，正在用这个主题搭建博客，有个问题，就是侧边栏哔哩哔哩，微信等图标连接都是您的，我又大部分没有，干脆想直接关闭，那该怎么配置呢？（先Starred再说）

@YunYouJun （抱歉打扰了繁忙的大佬）

**Comment by YunYouJun** [comment]
见[主题文档 ｜ 社交图标](https://yun.yunyoujun.cn/guide/config.html#%E7%A4%BE%E4%BA%A4%E5%9B%BE%E6%A0%87)。

**Comment by QrSn114** [comment]
谢谢大佬
